src/lib/db.js

```diff
@@ -969,7 +1029,9 @@ export default class db {
       });
     });
   }
-
+  
+  // Refactoring suggestion: rename to `addAuthor`
+  // to follow the style of the rest of the code
```

Comment:
Sure, you're welcome to do that. Quite a bit of the code needs some refactoring as too much of it was written under the pressure of a deadline a bit like now ;)